private.key

```diff
@@ -1,51 +0,0 @@
------BEGIN RSA PRIVATE KEY-----
```

Comment:
I hope you are not using these anywhere as they are public knowledge now :smile: